README.rst

```diff
@@ -60,11 +60,11 @@ You're done!
 Development
 -----------
 
-First, `install conda <http://conda.pydata.org/docs/install/quick.html>`_ and run
+First, `install conda <http://conda.pydata.org/docs/install/quick.html>`_ and run::
 
     $ conda install numpy scipy pytables
 
-After this, SAPPHiRE should work out of the box:
+After this, SAPPHiRE should work out of the box:::
```

Comment:
Yeah, I wanted one of the colons to be, you know, a colon. :P Oh, well. Making one more change then. :)